packages/jaeger-ui/config-overrides-antd-vars.less

```diff
@@ -22,3 +22,7 @@
 
 // Table
 @table-row-hover-bg:#e5f2f2;
+
+// Font
+// see https://github.com/ant-design/antd-init/tree/master/examples/local-iconfont
```

Comment:
That repo appears to be archived. What is the risk of using it?

Reply:
There is no risk because the fonts were included in this MR. I've only placed that comment to hint from where the fonts came from.

And from what I understood, it was only archived because newer versions of antd library no longer use the cdn.